bids/layout/config/bids.json

```diff
@@ -112,8 +112,8 @@
     ],
 
     "default_path_patterns": [
-        "sub-{subject}[/ses-{session}]/{datatype<anat>|anat}/sub-{subject}[_ses-{session}][_acq-{acquisition}][_ce-{ceagent}][_rec-{reconstruction}][_part-{part}]_{suffix<T1w|T2w|T1rho|T1map|T2map|T2star|FLAIR|FLASH|PDmap|PD|PDT2|inplaneT[12]|angio>}{extension<.nii|.nii.gz|.json>|.nii.gz}",
-        "sub-{subject}[/ses-{session}]/{datatype<anat>|anat}/sub-{subject}[_ses-{session}][_acq-{acquisition}][_ce-{ceagent}][_rec-{reconstruction}][_mod-{modality}]_{suffix<defacemask>}{extension<.nii|.nii.gz|.json>|.nii.gz}",
+        "sub-{subject}[/ses-{session}]/{datatype<anat>|anat}/sub-{subject}[_ses-{session}][_task-{task}][_acq-{acquisition}][_ce-{ceagent}][_rec-{reconstruction}][_run-{run}][_part-{part}]_{suffix<T1w|T2w|T1rho|T1map|T2map|T2star|T2starw|FLAIR|FLASH|PDmap|PD|PDw|PDT2|inplaneT[12]|angio>}{extension<.nii|.nii.gz|.json>|.nii.gz}",
```

Comment:
```suggestion
        "sub-{subject}[/ses-{session}]/{datatype<anat>|anat}/sub-{subject}[_ses-{session}][_task-{task}][_acq-{acquisition}][_ce-{ceagent}][_rec-{reconstruction}][_run-{run}][_part-{part}]_{suffix<T1w|T2w|T1rho|T1map|T2map|T2starmap|T2star|T2starw|FLAIR|FLASH|PDmap|PD|PDw|PDT2|inplaneT[12]|angio>}{extension<.nii|.nii.gz|.json>|.nii.gz}",
```

Reply:
You prefer to have it as a duplicate in the anat pattern even if it is already there in one of the qmri anat pattern ?

https://github.com/bids-standard/pybids/blob/c756db3056241c3c9d496d7759afd041916dd2b7/bids/layout/config/bids.json#L144

Reply:
Ah, well we have duplicates for T1map and T2map and PDmap as well, then.

Reply:
oops I had not realised

OK let me try to clean those up